scripts/wipe_sensitive_info_db.py

```diff
@@ -30,7 +37,10 @@ def main(argv):
       "WARNING: THIS WILL WIPE THE DATABASE IN-PLACE. CONTINUE?", err=True
     ):
       return
-  wipe_db_path(FLAGS.filename, FLAGS.keep_beds)
+  wipe_db_path(
+    FLAGS.filename, FLAGS.keep_beds, FLAGS.reset_admin, FLAGS.admin_email,
+    os.environ.get("ICUBAM_ADMIN_PASS", FLAGS.admin_pass)
```

Comment:
This doesn't seem to be documented in the script help.

Reply:
I'm not sure how the script help works -- isn't it autogenerated? Or did you mean that the env variable override is not documented?